Update AWS best practice page (#736)
smartcontractkit/documentation: docs/Node Operators/best-practices-aws.md

The data from both the Chainlink node and the Ethereum client is stored in the PostgreSQL database. The Aurora database cluster is fault tolerant by design and can handle an Availability Zone failure without any loss of data. There might be only a brief interruption of service with an automatic failover, which helps to reduce the Recovery Point Objective (RPO) and Recovery Time Objective (RTO).

Management of AWS service limits are not required for proper disaster recovery. The Quick Start deployment is configured with high availability in mind.

## Troubleshooting and Design Practices

### AWS Certificate Manager

> In the AWS console, if the **SSL certificate with AWS Certificate Manager** is set to `false`, we can leave **arn:aws:acm:region:account-id:certificate** as is. However, if you would like your node to be external facing and be reached by the internet, you will need to enable this feature and set it to `true`.

Reviewer: ```suggestion
In the AWS console, if the **SSL certificate with AWS Certificate Manager** is set to `false`, you can leave **arn:aws:acm:region:account-id:certificate** as is. However, if your node is external facing and must be reached by the internet, enable this feature and set it to `true`.
```

![AWS Certificate Manager](/images/node-operators/aws-certificate-manager.png)

### Amazon Devops Guru

> In the AWS console, if the **Amazon DevOps Guru** is set to `false`, we can leave **AdministerAccountId** and **EmailAddress** as is. 

Reviewer: ```suggestion
In the AWS console, if the **Amazon DevOps Guru** is set to `false`, you can leave **AdministerAccountId** and **EmailAddress** as is. 
```
